### ISSUE TITLE
[Grouping Accessibility Metadata] missing metadata

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#157]
This issue is part of the wider work presented in the issue #144 _Proposal for Grouping Accessibility Metadata into Categories_.

By cross-checking, I realized some metadata that is not present in the grouping file:

- `accessibilityFeature = pageBreakMarkers` (perhaps the pagination source should be included as well?)
- `accessibilityHazard = noFlashingHazard`
- `accessibilityHazard = noMotionSimulationHazard`
- `accessibilityHazard = noSoundHazard`
- `accessibilityHazard = unknown`

**Comment by gregoriopellegrino** [comment]
As decided in the [April 13 call](https://github.com/w3c/publ-a11y/wiki/Meeting-minutes-of-Publishing-CG-Accessibility-Task-Force#minutes-of-publishing-cg-accessibility-tf-meeting-held-on-april-13-2023-at-1400-utc) I've added these metadata in the spreadsheet.
